spec/controllers/ownerships_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```

Comment:
We should add tests to Market functionality, but I guess we can do this later (or maybe not? :))